geoextract/__init__.py

```diff
@@ -594,7 +595,7 @@ class BasicNormalizer(Normalizer):
     '''
 
     def __init__(self, to_ascii=True, rejoin_lines=True, remove_hyphens=True,
-                 remove_specials=True, subs=None, stem=None):
+                 remove_specials=True, substitutions=None, stem=None):
```

Comment:
I agree that `substitutions` is the better name. However, given the method's documentation the benefit of the change is small and does IMO not justify breaking backwards-compatibility by changing the public interface of the class.